docs/send-sms-explanation-outgoing-vs-incoming.md

```diff
@@ -0,0 +1,19 @@
+---
+title: SMS direction OUTGOING vs INCOMING
+topic: Send SMS
+type: explanation
+---
+
+# SMS direction OUTGOING vs INCOMING
+
+You can send an SMS with a `Direction` either as `OUTGOING` or `INCOMING`.
+
+A traditional SMS sent from one of your subscribers to another subscriber passes through the system two times. One time as `OUTGOING` and once as `INCOMING`. With the correct permissions it is possible to omit the outgoing SMS if you are sending to one of your subscribers. What to use depends on what kind of scenario you're facing.
```

Comment:
I think it only passes through the system twice if both the sender and receiver are subscribers of us.
Outgoing means that the message is going from one of our subscribers.
Incoming means the message is going to one of our subscribers.

Reply:
Good point, I'll change it.

Reply:
Changed. Better?

Reply:
Changed. Better?